crawler/twitter.js

```diff
@@ -209,13 +215,28 @@ twitter.crawlAll = function(full) {
         });
     });
 };
+}
 
 twitter.streamTweets = function(query) {
+    var output = '';
+    for (var i = 0; i < query.length; i++) {
+        output += query[i].name + ' , ';
+    }
+    output = "'" + output.substring(0, output.length - 2) + "'";
+    console.log("streaming...");
     client.stream('statuses/filter', {
-        track: query
+        track: output
     }, function(stream) {
-        stream.on('data', twitter.saveTweet);
-
+        stream.on('data', function(data) {
+            var lcString = util.inspect(data.text).toLowerCase();
+            for (var i = 0; i < query.length; i++) {
+                var lcSearchString = query[i].name.toLowerCase();
```

Comment:
I'm also not so sure if it is efficient enough to compare against > 2000 character in a linear way. I'd expect quite a lot of tweets during / after the episodes.
The streaming also doesn't allow "strict matching" as it seems. So we will also get a lot of strings which don't match anything.

If we are too slow Twitter will disconnect us. The reconnect logic is also currently missing.